.gitignore

```diff
@@ -0,0 +1,240 @@
+
+# Visual Studio profiler
+*.psess
+*.vsp
+*.vspx
+
+# Guidance Automation Toolkit
+*.gpState
+
+# ReSharper is a .NET coding add-in
+_ReSharper*/
+*.[Rr]e[Ss]harper
+
+# TeamCity is a build add-in
+_TeamCity*
+
+# DotCover is a Code Coverage Tool
+*.dotCover
+
+# NCrunch
+*.ncrunch*
+.*crunch*.local.xml
+_NCrunch*
+
+# Installshield output folder
+[Ee]xpress/
+
+# DocProject is a documentation generator add-in
+DocProject/buildhelp/
+DocProject/Help/*.HxT
+DocProject/Help/*.HxC
+DocProject/Help/*.hhc
+DocProject/Help/*.hhk
+DocProject/Help/*.hhp
+DocProject/Help/Html2
+DocProject/Help/html
+
+# Click-Once directory
+publish/
+
+# Publish Web Output
+*.Publish.xml
+*.pubxml
+
+# NuGet Packages Directory
+## TODO: If you have NuGet Package Restore enabled, uncomment the next line
+packages/
+
+# Windows Azure Build Output
+csx
+*.build.csdef
+
+# Windows Store app package directory
+AppPackages/
+
+# Others
+sql/
+*.Cache
+ClientBin/
+[Ss]tyle[Cc]op.*
+~$*
+*~
+*.dbmdl
+*.[Pp]ublish.xml
+*.pfx
+*.publishsettings
+
+# RIA/Silverlight projects
+Generated_Code/
+
+# Backup & report files from converting an old project file to a newer
+# Visual Studio version. Backup files are not needed, because we have git ;-)
+_UpgradeReport_Files/
+Backup*/
+UpgradeLog*.XML
+UpgradeLog*.htm
+
+# SQL Server files
+App_Data/*.mdf
+App_Data/*.ldf
+
+# =========================
+# Windows detritus
+# =========================
+
+# Windows image file caches
+Thumbs.db
+ehthumbs.db
+
+# Folder config file
+Desktop.ini
+
+# Recycle Bin used on file shares
+$RECYCLE.BIN/
+
+# Mac crap
+.DS_Store
+
+# ==================================================
+# Other common languages and tools
+# ==================================================
+
+# Jekyll generated site
+_site/
+
+# Node
+npm-debug.log
+#node_modules
+
+# Xcode
+*/build/*
+*.pbxuser
+!default.pbxuser
+*.mode1v3
+!default.mode1v3
+*.mode2v3
+!default.mode2v3
+*.perspectivev3
+!default.perspectivev3
+xcuserdata
+*.moved-aside
+DerivedData
+.idea/
+*.hmap
+*.xccheckout
+
+# Ruby
+*.gem
+*.rbc
+.bundle
+.config
+coverage
+InstalledFiles
+lib/bundler/man
+pkg
+rdoc
+spec/reports
+test/tmp
+test/version_tmp
+tmp
+
+# Rails
+*.sassc
+.sass-cache
+capybara-*.html
+.rspec
+.rvmrc
+/vendor/bundle
+/log/*
+/tmp/*
+/db/*.sqlite3
+/public/system/*
+/coverage/
+/spec/tmp/*
+**.orig
+rerun.txt
+pickle-email-*.html
+config/initializers/secret_token.rb
+
+# YARD artifacts
+.yardoc
+_yardoc
+
+# Android project (using Eclipse)
+/source/.settings/org.eclipse.jdt.core.prefs
+/source/bin/
+/source/gen/
+
+WebRole/App/main-built.js
+WebRole/DCATrace*
+WebRole/logs/*
+Nuget/**
+LocationTracking/Deploy/**
+IdentityServer/Deploy/**
+InternalRest/Deploy/**
+.vs/config/applicationhost.config
+nCrunchTemp*
+LocationTracking/App_Data/**
+IdentityServer/App_Data/**
+InternalRest/App_Data/**
+CJISReturnBroker/App_Data/**
+Dataonix_Audit/Dataonix_Audit.jfm
+Dataonix_Data/Dataonix_Data.jfm
+Dataonix_System/Dataonix_System.jfm
+*.playlist

```

Comment:
Please remove this last group, these do not belong in here.

Also, the ones that don't belong to .NET code... is there any benefit in having them? I think we should remove them, but it's not a strong preference. I'm thinking that we may end up generating a file that somehow is named in the same way and we may have confused developers because the repository will not track changes for them. Everything related to VS and .NET, completely agreed.